mon-pix/app/templates/components/signin-form.hbs

```diff
@@ -14,36 +14,38 @@
   </div>
 
   {{#if this.hasFailed}}
-    <div class="sign-form__notification-message sign-form__notification-message--error" aria-live="polite">
+    <div class="sign-form__notification-message sign-form__notification-message--error" role="alert">
       {{this.errorMessage}}
     </div>
```

Comment:
**suggestion**
Si c'est un message d'erreur, je crois que c'est mieux niveau a11y de le mettre dans une balise `<label>` avec un attribut `for` relié au champ en erreur (ici ça pourrait être sur le Form entier ? à vérifier).
L'avantage c'est que pour un lecteur d'écran il pourra lui dire que c'est un message en lien avec tel autre champ.